public class exception 
{

	public static void main(String[] args) 
	{
		// TODO Auto-generated method stub
		int a=10,b=0;
		int c[]=new int[5];
		try
		{
			c[2]=100;
			System.out.println(c[2]);
			System.out.println(a/b);
		}
		
		catch(ArrayIndexOutOfBoundsException|ArithmeticException e)
		{
			System.out.println(e);
		}
	}
}
